client/app/helpers/format-phone.js

```diff
@@ -0,0 +1,13 @@
+import { helper } from '@ember/component/helper';
+
+export function formatPhone(params) {
+  const [phone] = params;
+
+  if (!phone) {
+    return phone;
```

Comment:
Wait, why return phone if it's undefined or null?

Reply:
Oh, is this just to prevent undefined/null from going through the String replace function?

Reply:
falsey-ness isn't just undefined null — a blank string is also falsey.